### PR TITLE
Dev/folia

### PR DESCRIPTION
**Pull request by duplexsystem**
Folia support

### CODE REVIEW

**Comment by duplexsystem** [comment]
Thank you @pontaoski for the initial work!

File: buildSrc/src/main/kotlin/DependencyConfig.kt [hunk 1]
```diff
@@ -48,6 +48,9 @@ fun Project.configureDependencies() {
         maven("https://jitpack.io") {
             name = "JitPack"
         }
+        maven("https://nexuslite.gcnt.net/repos/other/") {
```
Comment (solonovamax) [inline buildSrc/src/main/kotlin/DependencyConfig.kt:51]: Is folialib published to maven central, or only here?

File: platforms/bukkit/build.gradle.kts [hunk 2]
```diff
@@ -3,7 +3,9 @@ plugins {
 }
 
 repositories {
-    maven("https://s01.oss.sonatype.org/content/repositories/snapshots/")
+    maven("https://s01.oss.sonatype.org/content/repositories/snapshots/") {
+        name = "sonatype"
```
Comment (solonovamax) [inline platforms/bukkit/build.gradle.kts:7]: very minor nitpick, not really an issue: "Sonatype" should be capitalized to fit with the implicit precedent set by the other repo names

**Review by solonovamax — APPROVED** [review]
LGTM

File: platforms/bukkit/build.gradle.kts (hunk 2 above)
Comment (duplexsystem) [reply platforms/bukkit/build.gradle.kts:7]: Will fix

File: buildSrc/src/main/kotlin/DependencyConfig.kt (hunk 1 above)
Comment (duplexsystem) [reply buildSrc/src/main/kotlin/DependencyConfig.kt:51]: Only here

File: platforms/bukkit/build.gradle.kts (hunk 2 above)
Comment (solonovamax) [reply platforms/bukkit/build.gradle.kts:7]: also same for "devmart-other", though unsure how that should be capitalized

though, you don't have to fix it, it's not a big deal

File: platforms/bukkit/common/src/main/resources/plugin.yml [hunk 3]
```diff
@@ -6,3 +6,4 @@ author: dfsek
 website: "@WIKI@"
 api-version: "1.13"
 description: "@DESCRIPTION@"
+folia-supported: true
```
Comment (solonovamax) [inline platforms/bukkit/common/src/main/resources/plugin.yml:9]: minor issue: this file should have a newline at the end (unsure if it does, bc I'm on gh mobile)

**Review by solonovamax — APPROVED** [review]

File: platforms/bukkit/common/src/main/resources/plugin.yml (hunk 3 above)
Comment (justaureus) [reply platforms/bukkit/common/src/main/resources/plugin.yml:9]: Looks like it does not

**Review by justaureus — APPROVED** [review]
Other than Solo's minor note everything looks good to me.

**Review by Astrashh — APPROVED** [review]

**Pull request merged**